src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/LinkDefendantToClaimTest.java

```diff
@@ -28,14 +29,15 @@ public void shouldReturn200HttpStatusAndUpdatedClaimWhenLinkIsSuccessfullySet()
 
     @Test
     public void shouldReturn404HttpStatusWhenClaimDoesNotExist() throws Exception {
-        long nonExistingClaimId = 900;
+        String nonExistingExternalId = "7d293143-b787-454f-aa8e-2fd69a209e52";
 
-        linkdefendantRequest(nonExistingClaimId, "1")
+        linkdefendantRequest(nonExistingExternalId, "1")
             .andExpect(status().isNotFound());
     }
 
-    private ResultActions linkdefendantRequest(Object claimId, String defendantId) throws Exception {
+    private ResultActions linkdefendantRequest(Object externalId, String defendantId) throws Exception {
```

Comment:
Please make method name properly capitalised to fix something we missed earlier.

Reply:
Please also make first arg of type String.

Reply:
hah, may as well